File: api/src/main/java/com/manageleaguefootball/demo/dto/TeamDTO.java
package com.manageleaguefootball.demo.dto;

import lombok.Data;
import org.springframework.web.multipart.MultipartFile;

@Data
public class TeamDTO {
    private String id;
    private String idSeason;
    private String name;
    private String phoneNumber;
    private String captainName;
    private String facebook;
    private String email;
    private int score;
    private int win;
    private int loss;
    private int draw;
    private int goalWin;
    private int goalLoss;
    private int difference;
    private String logoUrl;
}

File: src/app/dto/TeamDTO.ts
export interface TeamDTO {
  id?: string;
  idSeason?: string;
  name?: string;
  phoneNumber?: string;
  captainName?: string;
  //point
  score?: number;
  // so tran thang
  win?: number;
  // so tran thua
  loss?: number;
  //so tran hoa
  draw?: number;
  // so ban thang
  goalWin?: number;
  // so ban thua
  goalLoss?: number;
  // hieu so
  difference?: number;
}
